rust_src/src/editfns.rs

```diff
@@ -1177,4 +1319,39 @@ pub fn rust_find_field(
     }
 }
 
+/// Delete the field surrounding POS.
+/// A field is a region of text with the same `field' property.
+/// If POS is nil, the value of point is used for POS.
+#[lisp_fn(min = "0")]
+pub fn delete_field(pos: LispNumber) {
+    let mut isize: beg;
+    let mut isize: end;
+    rust_find_field(Some(pos), Qnil, Qnil, &beg, Qnil, &end);
+    if beg != end {
+        unsafe { del_range(beg, end) };
+    }
+}
+
+/// Return the contents of the field surrounding POS as a string.
+/// A field is a region of text with the same `field' property.
+/// If POS is nil, the value of point is used for POS.
+#[lisp_fn(min = "0")]
+pub fn field_string(pos: LispNumber) -> LispObject {
+    let mut isize: beg;
+    let mut isize: end;
```

Comment:
It looks like `rust_find_field` could return a tuple instead of using C style pointers to handle multiple returns.

Reply:
Yep, I made a PR based on this one. #1070.

I pulled the 3 users of find_field from this PR into it. Now there is no C version of the function and the pointers are gone.